src/components/navbar/Navbar.module.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../utils/variables';
+$tiny: 348px;
```

Comment:
Could this be put into variables? :)

Reply:
Sure, I didn't know why those specific size variables are up there and didn't think they wanted this kinda edge case up there. If someone later tries to round it to 350px it won't leave the nav in the best situation, although it is only 2 pixels of awkward range.

Reply:
nevermind, I just rounded it anyway, It isn't too bad. :]